news/_posts/releases/2020-11-18-prestashop-1-7-7-0-available.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 is available"
+subtitle: "You waited for it, here it is!"
+date:   2020-11-18 16:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/11/build_and_mail_final.jpg
+icon: icon-leaf
+tags:
+ - version
+ - 1.7
+ - 1.7.7.x
+ - minor
+ - releases
+---
+
+We are happy to announce PrestaShop 1.7.7.0 is officially available!
+
+![1.7.7.0 is available!](/assets/images/2020/11/build_and_mail_final.jpg)
+
+This release is our biggest yet since 1.7.1.0. More than one year of development, two betas, one release candidate Was it worth it? We absolutely think so, and hope that you will enjoy using it.
```

Comment:
```suggestion
This release is our biggest yet since 1.7.1.0. More than one year of development, two betas, one release candidate. Was it worth it? We absolutely think so and hope that you will enjoy using it.
```